python/common_modules_cff.py

```diff
@@ -117,6 +117,38 @@
     ) * probeMuonsIsoValueMaps
 )
 
+muonMiniIsoCharged = cms.EDProducer("MuonMiniIso",
+    probes = cms.InputTag("probeMuons"),
+    pfCandidates = cms.InputTag("pfAllChargedHadronsPFBRECO"),
+    dRCandProbeVeto = cms.double(0.0001),
+    CandPtThreshold = cms.double(0.0),
+    CandPdgId = cms.string('h'), # 'h','gamma','h0' or leave empty for all
+)
+
+muonMiniIsoPUCharged = cms.EDProducer("MuonMiniIso",
+    probes = cms.InputTag("probeMuons"),
+    pfCandidates = cms.InputTag("pfPileUpAllChargedParticlesPFBRECO"),
+    dRCandProbeVeto = cms.double(0.0001),
+    CandPtThreshold = cms.double(0.0),
+    CandPdgId = cms.string('h'), # 'h','gamma','h0' or leave empty for all
```

Comment:
the code looks simpler if I don't have to differentiate between PU (where you don't have e/mu) and noPU where you have e/mu and you have to remove them by hand)
https://github.com/cms-analysis/MuonAnalysis-TagAndProbe/pull/8/files#diff-d395ddc96a6c8f366c8e9974d3af9340R123
if you insist I will change it...